slider/styles.css

```diff
@@ -1,18 +1,47 @@
-body, .main {
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400&display=swap');
+
+/* Color variables */
+:root {
```

Comment:
nice! in one of the next classes, please explain to the group what you did here :)